src/components/SearchOverlay/SearchOverlay.js

```diff
@@ -5,8 +5,6 @@ import { connect } from 'react-redux';
 import onClickOutside from 'react-onclickoutside';
 
 import Icon from 'components/Icon';
-import Tooltip from 'components/Tooltip';
-import Input from 'components/Input';
```

Comment:
Do we want to use the toolkit for the `PrintModal` as well? That component uses `Input` as well.

Reply:
Yeah I'll do that one next, just slightly more complexity with it lining up properly